### ISSUE TITLE
varnishd -I clifile not executing last line if missing final line break

### ISSUE THREAD

**Issue by prodigion** [varnishcache/varnish-cache#3995]
### Expected Behavior

Following https://info.varnish-software.com/blog/one-vcl-per-domain

My start.cli file did not have a final line break after the last line.

```vcl.load vcl-w-orig /etc/varnish/w.vcl
vcl.load vcl-x-orig /etc/varnish/wiki.vcl
vcl.label label-w vcl-w-orig
vcl.label label-x vcl-x-orig
vcl.load vcl-root-orig /etc/varnish/root.vcl
vcl.use vcl-root-orig
```

After adding the line break, it worked correctly
```
varnishadm vcl.list
available warm   warm       0  vcl-w-orig     <-  (1 label)
available warm   warm       0  vcl-x-orig     <-  (1 label)
available label  warm       0  label-w        ->  vcl-w-orig (1 return(vcl))
available label  warm       0  label-x        ->  vcl-x-orig (1 return(vcl))
active    auto   warm       0  vcl-root-orig
```

### Current Behavior

It loaded the vcl-root-orig, but did not use it

```
varnishadm vcl.list
active    warm   warm       0  vcl-w-orig     <-  (1 label)
available warm   warm       0  vcl-x-orig     <-  (1 label)
available label  warm       0  label-w        ->  vcl-w-orig (1 return(vcl))
available label  warm       0  label-x        ->  vcl-x-orig (1 return(vcl))
available cold   cold       0  vcl-root-orig
```

### Context

All cache went through vcl-w-orig instead of respecting the if statement.

### Varnish Cache version

varnishd (varnish-6.2.1 revision 9f8588e4ab785244e06c3446fe09bf9db5dd8753)

### Operating system

Ubuntu 20.04.6 LTS

**Comment by gquintard** [comment]
Thanks @prodigion! As discussed on discord, I think we can easily add a line break automatically as that would be what most people would expect, but let's see what the core team thinks

**Comment by Dridi** [comment]
Not an opinion, but here is what the documentation says:

> Every line in the file, including the last line, must be terminated by a newline or carriage return.

https://varnish-cache.org/docs/7.4/reference/varnishd.html#cli-command-file

**Comment by bsdphk** [comment]
We insist on the NL because incomplete CLI commands are dangerous, in particular the ``<< nonce`` form of commands.

I dont think optimistically appending a NL and hoping for the best is a good idea.

In general CLI sources are supposed to have somebody at the other end awaiting the CLI response, so issuing a diagnostic if they leave an incomplete command we do not execute is not needed, except for the case of ``-I cli_file``.

I'm working on a patch to get diagnostics in the ``-I cli_file`` case.

**Comment by bsdphk** [comment]
Fixed in cd2cfbd522
